lib/collection/src/shards/transfer/mod.rs

```diff
@@ -43,9 +43,9 @@ pub struct ShardTransfer {
     /// If this flag is true, this is a replication related transfer of shard from 1 peer to another
     /// Shard on original peer will not be deleted in this case
     pub sync: bool,
-    /// Method to transfer shard with. `None` to choose automatically.
+    /// Method to transfer shard with.
     #[serde(default)]
-    pub method: Option<ShardTransferMethod>,
```

Comment:
Please note.

By the point this is written to disk, it should always have a value. If this assumption is correct, then there should be no problem here.

However, if by some reason it didn't (maybe some earlier version did not have this), `ShardTransferMethod::default()` will be set automatically here, overriding the collection's default method set in the config.